react/tic-tac-toe/src/app/screens/Login/layout.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import PropTypes from 'prop-types';
+
+import styles from './styles.module.scss';
+import LoginInput from './components/LoginInput';
+import { hasLenght, isRequired, isValidEmail } from './Validations';
+
+function Login({ handleSubmit }) {
+  return (
+    <div className={styles.loginContainer}>
+      <h1 className={styles.loginTitle}>Login</h1>
+      <form className={styles.loginForm} onSubmit={handleSubmit}>
+        <label className={styles.loginDetail}>Email</label>
+        <Field
+          name="email"
+          type="text"
+          label="Email"
+          component={LoginInput}
+          validate={[isValidEmail, isRequired]}
+        />
+        <label className={styles.loginDetail}>Password</label>
```

Comment:
Why not setting this in your components ? You are already sending label as a prop

Reply:
didn't realise i had two labels until this review, thanks, id rather delete the label prop from LoginInput.

Reply:
I rather you add the label to the LoginInput and receive the prop, I think most inputs have a label. Also add the htmlFor prop so clicking the label focuses the input, that's a good practice